addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3413,15 +3413,21 @@ export class OdooEditor extends EventTarget {
             } else if (['insertText', 'insertCompositionText'].includes(ev.inputType)) {
                 // insertCompositionText, courtesy of Samsung keyboard.
                 const selection = this.document.getSelection();
-                // Detect that text was selected and change behavior only if it is the case,
-                // since it is the only text insertion case that may cause problems.
-                const wasTextSelected = anchorNodeOid !== focusNodeOid || anchorOffset !== focusOffset;
+                // Detect that text was selected accross nodes and change the
```

Comment:
You say in the commit message that it's only needed if the selection spans multiple "element nodes". Maybe be more specific here (just "element" is enough, since an element is a node), otherwise it might seem at first glance to the reader like multiple text node could do.